ste/mgr-JobPartTransferMgr.go

```diff
@@ -90,6 +90,7 @@ type IJobPartTransferMgr interface {
 }
 
 type TransferInfo struct {
+	JobID 				   common.JobID
```

Comment:
If I understood correctly, duplicating the jobID info here would not bloat the plan files, but what happens if there's a resume?

Reply:
This is a in-memory struct created from plan file. It'll be recreated on resume. Nothing from here is flushed to disk.